openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -7661,6 +7672,7 @@ def main():
     parser.add_argument('--output_h5', action='store_true', help='.h5 output switch')
     parser.add_argument('--output_weight_and_json', action='store_true', help='weight of h5 and json output switch')
     parser.add_argument('--output_pb', action='store_true', help='.pb output switch')
+    parser.add_argument('--dont_save_keras_model', action='store_true', help='use TFLiteConverter.from_concrete_functions to avoid saving Keras model to disk during TFLite conversion')
```

Comment:
To be precise, `dont_convert_from_keras_model` is more correct in meaning than `dont_save_keras_model`, since we are not outputting keras_model, but only reading keras model objects.